arbitrum-docs/launch-orbit-chain/orbit-sdk/how-tos/configuring-orbit-chain-with-sdk.md

```diff
@@ -0,0 +1,129 @@
+---
+title: 'How to configure an Orbit chain using the Orbit SDK'
+sidebar_label: 'Orbit Chain Configuration'
+description: 'How to configure an Orbit chain using the Orbit SDK '
+author: Mehdi Salehi
+sme: Mehdi Salehi
+target_audience: 'Developers deploying and maintaining Orbit chains.'
+sidebar_position: 6
+---
+
+This guide will explore the essentials of configuring your Orbit chain. Configuration includes a range of settings from the parent chain and node configuration to specific child chain parameter configurations. Our focus here is on child chain configuration, guiding you through the necessary adjustments to customize your chain's operation.
+
+In the introductory section, we outlined three primary configurations required for setting up and managing an Orbit chain. These configurations play a crucial role in ensuring the efficient operation of your chain. Let's delve into each type:
+
+## 1. Parent Chain Configuration
```

Comment:
```suggestion
## 1. Parent chain configuration
```

nit - sentence-case titles of all kinds ([ref](https://docs.arbitrum.io/for-devs/contribute#style-conventions))

Reply:
Missed this one entirely, thanks.